src/foam/lib/json/Outputter.java

```diff
@@ -96,17 +97,33 @@ protected void outputNull() {
   }
 
   public void outputString(String s) {
-    writer_.append("\"");
-    writer_.append(escape(s));
-    writer_.append("\"");
+    if ( multiLineOutput_ && s.indexOf('\n') >= 0 ) {
+      writer_.append("\n  \"\"\"\n    ");
+      s = ltrim(s, 4);
+      writer_.append(escapeMultiline(s));
+      writer_.append("\n  \"\"\"");
+    }
+    else {
+      writer_.append("\"");
+      writer_.append(escape(s));
+      writer_.append("\"");
+    }
   }
 
   public String escape(String s) {
-    return s
-      .replace("\\", "\\\\")
-      .replace("\"", "\\\"")
-      .replace("\t", "\\t")
-      .replace("\n","\\n");
+    return s.replace("\\", "\\\\")
+            .replace("\"", "\\\"")
+            .replace("\t", "\\t")
+            .replace("\n","\\n");
+  }
+
+  public String escapeMultiline(String s) {
+    s = s.replace("\\", "\\\\")
+         .replace("\\\"", "\"")
```

Comment:
I don't think this one line is necessary, we don't need to unescape escaped quotation marks in the string.

Reply:
we don't, but since the string is multiline the quatation marks don't need to be escaped and it makes things harder to read

Reply:
this simply cleans up the formatting to make it more readable

Reply:
I still don't think we should be doing this, they may be pre-escaped for a reason.  If my script reads

```
print("\\" hello\\"")
```
this escaping will turn it in to
```
print("" hello"");
```
which is no longer valid code.

Additionally we shouldn't do unescaping in a method named "escape" ;)